e2e_test/batch/basic/union.slt.part

```diff
@@ -69,9 +69,55 @@ NULL
 statement error
 select null union all select null select union 1
 
+query II
+select * from t1 union all corresponding select * from t2 order by v1
+----
+1 3
+1 3
+
+query II
+select * from t1 union corresponding select v4, v3 as v1 from t2 order by v1
+----
+1 3
+2 3
+
+query II
+select * from t1 union all corresponding by (v4, v1) select * from t2
+----
+3 1
+3 1
+
+query II
+select * from t1 union corresponding by (v4) select * from t2
+----
+3
+
+statement error
+select * from t1 union corresponding by (vxx) select * from t2
+----
+db error: ERROR: Failed to run the query
+
+Caused by:
+  Invalid input syntax: Every column name in the corresponding column list shall be a column name of both left and right side query in a UNION operation. Column name `vxx` in the corresponding column list is not found in a query column list: ("v1", "v2", "v4").
```

Comment:
Sorry to push back again, but it still seems too long and verbose to me. 

Besides, `("v1", "v2", "v4")` looks confusing, as it's not specified explicitly in the query and we don't know where it's from...


What about

```suggestion
  Invalid input syntax: Column name `vxx` in CORRESPONDING BY is not found in a side of the UNION operation. It shall be included in both sides. 
```

Reply:
Sure I'm okay with this.